### ISSUE TITLE
create a list of topics for discussion

### ISSUE THREAD

**Issue by ckundo** [ckundo/a11y-workshop#5]
What aspects of accessible development do we want to cover? What can we skip?

**Comment by ckundo** [comment]
Some ideas:
- build a static site using bootstrap with accessibility plugins
- test a site for color contrast failures
- demo color contrast test in Sketch
- screen reader demo
- demo accessible static site (paciellogroup.com)
- demo accessible single page app (http://dylanb.github.io/todomvc/index.html#/)
